### ISSUE TITLE
Support linking to other slides

### ISSUE THREAD

**Issue by MartenBE** [MartenBE/mkslides#6]
If not absolute path, and ends on `.md`: replace the extension with `.html`

**Comment by MartenBE** [comment]
Perhaps something similar to the `nav` option in `mkdocs` would be great.

**Comment by jaywhj** [comment]
Yes, this is necessary. If the markdown document is associated with other markdown files, the associated md files will be copied to the built directory during the build. This is actually not necessary. You only need to change the suffix to html.

**Comment by MartenBE** [comment]
This is tracked in https://github.com/MartenBE/mkslides/pull/34

**Comment by MartenBE** [comment]
Added in https://github.com/MartenBE/mkslides/pull/34
